airbyte-integrations/connectors/destination-gcs/src/test-integration/java/io/airbyte/integrations/destination/gcs/GcsAvroDestinationAcceptanceTest.java

```diff
@@ -114,4 +120,28 @@ public void convertDateTime(ObjectNode data, Map<String, String> dateTimeFieldNa
     }
   }
 
+  @Override
+  protected void deserializeNestedObjects(final List<AirbyteMessage> messages, final List<AirbyteRecordMessage> actualMessages) {
+    for (final AirbyteMessage message : messages) {
+      if (message.getType() == Type.RECORD) {
+        final var iterator = message.getRecord().getData().fieldNames();
+        while (iterator.hasNext()) {
+          final var fieldName = iterator.next();
+          if (message.getRecord().getData().get(fieldName).isContainerNode()) {
+            message.getRecord().getData().get(fieldName).fieldNames().forEachRemaining(f -> {
+              final var data = message.getRecord().getData().get(fieldName).get(f);
+              final var wrappedData = String.format("{\"%s\":%s,\"_airbyte_additional_properties\":null}", f,
+                  dateTimeFieldNames.containsKey(f) || !data.isTextual() ? data.asText() : StringUtils.wrap(data.asText(), "\""));
+              try {
+                ((ObjectNode) message.getRecord().getData()).set(fieldName, new ObjectMapper().readTree(wrappedData));
+              } catch (final JsonProcessingException e) {
+                e.printStackTrace();
+              }
+            });
+          }
+        }
+      }
+    }
+  }
+
```

Comment:
These seemed to be missing from https://github.com/airbytehq/airbyte/pull/9816 for these tests to pass...

Same thing in all the other acceptanceTest files from this PR